src/modules/esl-random-text/core/esl-random-text.ts

```diff
@@ -0,0 +1,116 @@
+import {ESLBaseElement} from '../../esl-base-element/core';
+import {attr, boolAttr} from '../../esl-utils/decorators';
+import {capitalize} from '../../esl-utils/misc/format';
+
+export class ESLRandomText extends ESLBaseElement {
+  public static override readonly is = 'esl-random-text';
+  public static readonly observedAttributes = ['paragraphs', 'words', 'shuffle'];
+
+  /** Last used word index in dictionary */
+  protected static pointer: number = -1;
+  /** Words dictionary to use in random text generation. */
+  public static DICTIONARY: string[] = [
+    'lorem', 'ipsum', 'dolor', 'sit', 'amet',
+    'consectetur', 'adipiscing', 'elit', 'curabitur',
+    'ultrices', 'et', 'mi', 'suscipit', 'eget', 'vulputate', 'ante',
+    'proin', 'vel', 'pretium', 'enim', 'vivamus', 'venenatis', 'eu',
+    'urna', 'tempor', 'blandit', 'nullam', 'pellentesque', 'metus',
+    'rhoncus', 'mauris', 'mollis', 'neque', 'sed', 'tincidunt', 'tellus',
+    'nunc', 'ac', 'nulla', 'ut', 'purus', 'etiam', 'id', 'dui', 'justo',
+    'sapien', 'scelerisque', 'viverra', 'ligula', 'aenean', 'porta',
+    'condimentum', 'nibh', 'dictum', 'congue', 'odio', 'facilisis',
+    'finibus', 'mattis', 'vehicula', 'lacinia', 'risus', 'placerat',
+    'augue', 'fringilla', 'at', 'facilisi', 'arcu', 'diam', 'laoreet'
+  ];
+  /** Number of words in sentence */
+  public static readonly WORDS_PER_SENTENCE = 10;
+
+  /** Choose words randomly from {@link DICTIONARY} rather than sequentially */
+  @boolAttr() public shuffle: boolean;
+  /** Maximum number of words in generated text */
+  @attr({parser: parseInt, defaultValue: NaN}) public words: number;
```

Comment:
@abarmina suggested a very reasonable idea:
- get rid of the `words` as currently they are not even works as expected
- make user able to pass float values to paragraph field
- make default paragraphs count - 1

As the result we do not have incompatible properties (words vs paragraphs), but all cases still covered including primary.